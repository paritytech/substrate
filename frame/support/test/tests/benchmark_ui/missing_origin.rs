use frame_benchmarking::v2::*;
#[allow(unused_imports)]
use frame_support_test::Config;

#[benchmarks]
mod benches {
	use super::*;

	#[benchmark]
	fn bench() {
		#[extrinsic_call]
		thing();
	}
}

fn main() {}
