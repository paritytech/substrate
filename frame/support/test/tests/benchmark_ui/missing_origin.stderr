error: Single-item extrinsic calls must specify their origin as the first argument.
  --> tests/benchmark_ui/missing_origin.rs:12:3
   |
12 |         thing();
   |         ^^^^^
